tests/app-template-minimal/package.json

```diff
@@ -6,41 +6,47 @@
   "repository": "",
   "license": "MIT",
   "author": "",
+  "type": "module",
+  "imports": {
+    "#/*": "./src/*",
+    "#config": "./src/config.js",
+    "#components/*": "./src/components/*"
+  },
   "exports": {
     "./tests/*": "./tests/*",
-    "./*": "./app/*"
+    "./*": "./src/*"
   },
   "scripts": {
     "build": "vite build",
+    "build:test": "NODE_ENV=development vite build --mode test",
```

Comment:
cross-env?

Reply:
I realized there's a better solution than this env var. The vite docs say you can create `.env.[mode]` and set `NODE_ENV` inside there. So if we make `.env.test` with `NODE_ENV=development` that should do it.